.github/workflows/ci-code.yml

```diff
@@ -99,7 +102,15 @@ jobs:
       env:
         AIIDA_TEST_PROFILE: test_aiida
         AIIDA_WARN_v3: 1
-      run: pytest --cov aiida --verbose tests -m 'not nightly'
+      # Python 3.12 has a performance regression when running with code coverage
+      # so run code coverage only for python 3.9.
+      run: pytest -v tests -m 'not nightly' ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
+
+    - name: Run verdi tests
+      run: |
+        verdi devel check-load-time
+        verdi devel check-undesired-imports
+        .github/workflows/verdi.sh
```

Comment:
Yeah, that's fair. I've put it back, but only with python 3.12, I don't see much benefit in running these with multiple Python version, in fact running it more just increases a chance of flukes, since one of the tests is literally measuring time to execute verdi. LMK what you think.

Reply:
That's fine